Sources/Confidence/FlagEvaluation.swift

```diff
@@ -87,6 +89,9 @@ extension FlagResolution {
                 } else {
                     // `null` type from backend instructs to use client-side default value
                     if parsedValue == .init(null: ()) {
+                        Task {
+                            await flagApplier?.apply(flagName: parsedKey.flag, resolveToken: self.resolveToken)
+                        }
```

Comment:
This is the actual fix. Repeating this code is not ideal, but I think it makes it more readable and less error prone in the future